api/src/main/java/jakarta/persistence/EntityListeners.java

```diff
@@ -27,17 +27,69 @@
  * entity or mapped superclass. This annotation may be applied 
  * to an entity class or mapped superclass.
  *
+ * <pre>
+ * &#64;Entity
+ * &#64;EntityListeners(BookObserver.class)
+ * class Book { ... }
+ * </pre>
+ *
  * <p>The specified entity listener classes may have callback
  * methods annotated {@link PrePersist}, {@link PreUpdate},
  * {@link PreRemove}, {@link PostPersist}, {@link PostUpdate},
- * and/or {@link PostRemove}.
+ * {@link PostRemove}, and/or {@link PostLoad}. A callback
+ * method defined by an entity listener class must have the
+ * signature {@code void method(E entity)} where {@code E} is
+ * a supertype of an entity class or mapped superclass to which
+ * the entity listener applies.
+ *
+ * <pre>
+ * class BookObserver {
+ *
+ *     &#64;PostPersist
+ *     void newBook(Book book) {
+ *         ...
+ *     }
+ *
+ * }
+ * </pre>
+ *
+ * <p>An entity listener class may have multiple callback
+ * methods for a given type of lifecycle event, but at most
+ * one callback method for a given type of event and given
+ * entity type.
```

Comment:
The entity type isn't always given because the given type can be a superclass instead.  If I understand the intent here, it should not be possible in the following example to have `@PostPersist void postPersist(Object o)` alongside the other `@PostPersist` methods because it would result in multiple PostPersist events for the same entity.

```suggestion
 * one callback method for a given type of event and entity
 * type.
```